client/commands/retro.js

```diff
@@ -0,0 +1,11 @@
+/* global commandFuncs, openFlexPanel, getServiceBaseURL, GAME */
+
+commandFuncs.retro = args => {
+  if (args.help || args._.length > 1) {
+    return  // handled by game-cli in server/index.js
+  }
+
+  const projectPath = args._.length === 1 ? `/${args._[0].replace('#', '')}` : ''
+  const flexPanelURL = `${getServiceBaseURL(GAME)}/retro${projectPath}#${Date.now()}`
```

Comment:
On second thought, `/retro/{name}` does seem better overall. Considering only the app routing, I don't like the idea of `/retro` and `/retro/{name}` loading the same project-specific component, but it leaves us most flexible to change the behavior for `/retro` in the future. Perhaps we can have `/retro` load either a list of project links that can point to `/retro/{name}` (and the project-specific UI). In the case of there being only 1 project found, we could also have `/retro` "redirect" to `/retro/{name}`